tests/functions/test_convert_stock.py

```diff
@@ -0,0 +1,7 @@
+from janitor import get_symbol
```

Comment:
Because this is a test for a finance-themed function, I think it should go under the finance tests instead.

Reply:
**Action item:** Please move this into the finance.py submodule.